website/docs/r/iot_dps.html.markdown

```diff
@@ -0,0 +1,70 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_iot_dps"
+sidebar_current: "docs-azurerm-resource-messaging-iot-dps-x"
+description: |-
+  Manages an IoT Device Provisioning Service.
+---
+
+# azurerm_iot_dps
+
+Manages an IoT Device Provisioning Service.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "resourceGroup1"
+  location = "West US"
+}
+
+resource "azurerm_iot_dps" "test" {
+  name                = "test"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  location            = "${azurerm_resource_group.test.location}"
+
+  sku {
+    name     = "S1"
+    tier     = "Standard"
```

Comment:
```suggestion
   tier     = "Standard"
```

Reply:
I don't think we need this one. This change changes the spaces from 4 to 3